docs/docs/getting-started/installation.md

```diff
@@ -0,0 +1,42 @@
+---
+sidebar_position: 1
+title: Installation
+---
+### Clone the project
+
+To get started, you'll need to clone the `noteable-io/kernels` project from Github.
+
+```sh
+git clone git@github.com:noteable-io/kernels
+cd kernels
+```
+
+### Requirements
+
+In order to build the various kernels provided at Noteable, you'll need a few system tools.
+
+- [Docker](https://docs.docker.com/engine/install/)
+- [Taskfile](https://taskfile.dev/installation/)
```

Comment:
Specify the versions for both Docker and Taskfile

Reply:
Again, there isn't really a _required_ version of these. Any recent version of Docker or Taskfile should be fine, and I don't think we need / want to come back and update this as newer versions are released. 

If we start running into build or dev issues because of open issues on either binary, then I think we can update this doc to temporarily pin them if we need to

Reply:
We should specify minimum version required to help get folks on the same page as we are to reduce dependency issues. We do it for CI already, so we should just link to the versions we use there for consistency